### PR TITLE
Fix missing electron veto, add W-like analysis, fix some bugs with helicity/angular coeffs, clip qcd scale weights, and bump narf to fix hist conversion

### PR DESCRIPTION
**Pull request by bendavid**
Theory uncertainties for W-like are still incomplete (no helicity-decorrelated scale variations or scetlib corrections yet)

### CODE REVIEW

File: scripts/histmakers/mz_wlike_with_mu_eta_pt.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+muon_prefiring_helper, muon_prefiring_helper_stat, muon_prefiring_helper_syst = wremnants.make_muon_prefiring_helpers(era = era)
+
+# TODO make this properly configurable for Z
+#scetlibCorr_helper = wremnants.makeScetlibCorrHelper()
+
+qcdScaleByHelicity_helper = wremnants.makeQCDScaleByHelicityHelper(is_w_like = True)
+axis_ptVgen = qcdScaleByHelicity_helper.hist.axes["ptVgen"]
+axis_chargeVgen = qcdScaleByHelicity_helper.hist.axes["chargeVgen"]
+
+wprocs = ["WplusmunuPostVFP", "WminusmunuPostVFP", "WminustaunuPostVFP", "WplustaunuPostVFP"]
```
Comment (kdlong) [inline scripts/histmakers/mz_wlike_with_mu_eta_pt.py:40]: Should probably be defined in some helper class

File: wremnants/include/theoryTools.h [hunk 2]
```diff
@@ -99,7 +99,8 @@ scale_tensor_t makeScaleTensor(const Vec_f &scale_weights) {
   res(2, 1) = scale_weights[7]; //mur=2.0 muf=1.0;
   res(2, 2) = scale_weights[8]; //mur=2.0 muf=2.0;
 
-  return res;
+  // clip large weights
+  return clip_tensor(res, thres);
```
Comment (kdlong) [inline wremnants/include/theoryTools.h:103]: An extra loop :o

Comment (bendavid) [reply wremnants/include/theoryTools.h]: "fixed" :P

**Comment by kdlong** [comment]
Feel free to split this stuff into multiple PRs next time :)

**Pull request merged**